deploy/docs/Installation_with_Helm.md

```diff
@@ -69,26 +69,10 @@ To customize your configuration, download the values.yaml file by running
 curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/v0.15.0/deploy/helm/sumologic/values.yaml
 ```
 
-NOTE: If you need to install the chart with a different release name or namespace you will need to override some configuration fields for both Prometheus and fluent-bit. We recommend using an override file due to the number of fields that need to be overridden. In the following command, replace the `<RELEASE-NAME>` and `<NAMESPACE>` variables with your values and then run it to download the override file with your replaced values:
+####To install the chart with a different release name or namespace:
```

Comment:
I think you need a space `#### To` for the header to be parsed correctly

Maybe we can get rid of this section altogether? since now there is nothing special about `name` or `namespace`, from the default install command the users would be able to tell that they could change the name or namespace parameters. What do we think?

Reply:
I think we should still keep the example command, as everywhere else we are using the default release name and namespace.